bin/run_controller

```diff
@@ -150,11 +152,11 @@ if [ $IMAGE == "forch/controller" ] && [[ $faucet_id =~ ^[0-9]+$ ]]; then
     data_intf_name=data_faucet_$faucet_id
     sudo ip link add $data_intf_name type veth peer name data0 netns $pid
     sudo ip link set $data_intf_name up
-    docker exec $CONTAINER ip link add link data0 name data0.272 type vlan id 272
-    docker exec $CONTAINER ip addr add 192.168.2.$faucet_id dev data0.272
+    docker exec $CONTAINER ip addr add 192.168.2.$faucet_id dev data0
     docker exec $CONTAINER ip link set data0 up
-    docker exec $CONTAINER ip link set data0.272 up
-    docker exec $CONTAINER ip route add 192.168.0.0/16 dev data0.272
+    docker exec $CONTAINER /usr/share/openvswitch/scripts/ovs-ctl start
+    docker cp misc/controller_go $CONTAINER:/usr/bin/controller_go 
+    docker exec $CONTAINER /usr/bin/controller_go
```

Comment:
controller_go should be this last thing in the Dockerfile, instead of faucet_go

Reply:
Changed. faucet_go isnt called any more.